src/helpers/GetTotalDonated.ts

```diff
@@ -32,13 +38,70 @@ export const getTotalDonated = async ({ address, networkID, provider }: IBaseAdd
 
     // Pull user's redeemable balance and add to amount redeemed
     const givingContract = new ethers.Contract(addresses[networkID].GIVING_ADDRESS as string, OlympusGiving, provider);
-    const redeemableBalance = await givingContract.redeemableBalance(address);
+    const redeemableBalance = await givingContract.totalRedeemableBalance(address);
 
     const totalDonated = totalRedeemed.add(redeemableBalance);
+    const totalDonatedAsSohm = await gohmContract.balanceFrom(totalDonated);
 
-    return ethers.utils.formatUnits(totalDonated, "gwei");
-  } else {
+    return ethers.utils.formatUnits(totalDonatedAsSohm, "gwei");
+  }
+};
+
+// Calculate total yield sent from a given user to a given recipient
```

Comment:
Do the `/**` style function comment, as VSCode (etc) will automatically recognise it, so that when you hover over the function call in another file, it’ll display the documentation.